INSTALL.md

```diff
@@ -199,8 +199,8 @@ Follow the steps below to solve this problem:
 
 Try to increase the memory allocated to Docker. If this doesn't work, you can reduce the number of threads in CMake build in `[Milvus root path]/core/build.sh`.
```

Comment:
Could you please also use **Try increasing** here? I just noticed this error. I might have overlooked this before.